### PR TITLE
Evapo Chart

### PR DESCRIPTION
**Pull request by anthowen**
## What's NEW
- Refresh data per 1 second
- Added chart using HighCharts library
- Added limit control to manage the number of dates to be shown on the chart
- Deployed to AWS S3
- Showed the data in a table
  **Build command**: `yarn build && yarn deploy`

### screenshot:
image

### CODE REVIEW

**Comment by anthowen** [comment]
@hwestbrook Tbh, I'm new to HighCharts library, so it took some time to replicate the demo graph exactly 😂 
Could you review the PR?

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                name: 'Temperature (F)',
+                data: data.temps,
+                yAxis: 1,
+                tooltip: {
+                  valueDecimals: 2
+                }
+              },{
+                name: 'Evapo (In)',
+                data: data.evapos,
+                yAxis: 0,
```
Comment (hwestbrook) [inline src/App.js:124]: Evapo should be on its own axis

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                    format: '{value} in',
+                  },
+                  title: {
+                    text: 'Evapotranspiration',
+                  },
+                },
+              ],
+              series: [{
+                name: 'Temperature (F)',
+                data: data.temps,
```
Comment (hwestbrook) [inline src/App.js:116]: Your temps on the graph should be converted to F. I would put the temp F on the same axis as solar radiation.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        const day = moment(date).format("MMM Do");
+
+        let url = `${API_URL}/${DEVICE_ID}?date=${date}&tzOffset=${TZ_OFFSET}&elevation=${elevation}&latitude=${LATITUDE}&Kc=${coef}`;
+
+        const response = await fetch(url);
+        const json = await response.json();
+
+        const { meanDailyAirTemperatureC, meanSolarRadiationMJ, avgWindSpeedMs, atmosphericPressue, netRadiation, evapotranspirationMM, evapotranspirationIN} = json;
+
+        setData({
```
Comment (hwestbrook) [inline src/App.js:48]: If we were doing this for our app, I would want you to figure out how to add data to chart instead of causing a redraw of chart.

Comment (anthowen) [reply src/App.js:48]: Sure. Redraw is also what I've been spending time during this task. I tried a solution using ref, not to change the data reference value, but only update its contents.  However, HighCharts itself was not able to identify the value change, and chart wasn't updating. (it looks like they don't do deep equal check, but only depends on reference check - isEqual).

Comment (anthowen) [reply src/App.js:48]: To be honest, I've already worked on this similar scenario for the data monitoring system, and I used `recharts` library. It detected the deep value change, and wasn't redrawing on live data show

**Comment by anthowen** [comment]
> @anthowen -- this looks good and is what I would expect for this task.
> 
> No need to follow up on the review comments below. I'm point out things we would do before releasing to our users, but since this won't be released, there is no reason to change.

Thanks for the review and feedback. Definitely, it would be in the best quality in case it was release :)

**Comment by anthowen** [comment]
> Closing this pull request. Thank you for your submission Anthony. I'll let you know about the position in the next week or two.

Alright. Thank you!

**Pull request closed without merging**